test/school_test.rb

```diff
@@ -45,11 +40,36 @@ def test_it_can_add_student_names
   end
 
   def test_it_can_calculate_end_time
-    skip
     school1 = School.new('9:00', 7)
     school2 = School.new('9:00', 3)
 
     assert_equal '16:00', school1.end_time
     assert_equal '12:00', school2.end_time
   end
+
+  def test_it_can_determine_if_school_is_full_time
+    school1 = School.new('9:00', 7)
+    school2 = School.new('9:00', 4)
+
+    assert_equal true, school1.is_full_time?
+    assert_equal false, school2.is_full_time?
+  end
+
+  def test_standard_student_name_will_capitalize_names
+    school = School.new('9:00', 7)
+
+    school.add_student_name('Aurora')
+    school.add_student_name('tim')
+    school.add_student_name('megan')
+
+    standard_student_names = ["Aurora", "Tim", "Megan"]
+
+    assert_equal standard_student_names, school.standard_student_names
+  end
+
+  def test_it_can_convert_end_time_to_clock_time
```

Comment:
It would be good to have a test for a school day that ends before noon, so that you could make sure that your method works for any end time!